config/initializers/her.rb

```diff
@@ -1,4 +1,7 @@
 require 'faraday_middleware/service_urls'
+require 'her/model/propogate_errors'
+
+Her::Model.include(Her::Model::PropogateErrors)
```

Comment:
```suggestion
Her::Model.include(Her::Model::PropagateErrors)
```

Reply:
Ha ha - my poor spelling - I'll get that fixed